.gitignore

```diff
@@ -2,4 +2,5 @@
 *.pyc
 .idea
 .env*
+celerybeat-schedule 
```

Comment:
This line is not needed, please remove it together with the `celerybeat-schedule` file